components/nav/NavFooter.vue

```diff
@@ -38,3 +43,23 @@ function toggleDark() {
     </div>
   </footer>
 </template>
+
+<style>
+  .beta-label {
+    width: 212px;
+    height: 70px;
+    background-color: var(--c-primary);
+    border: 2px solid var(--c-primary);
+    border-radius: 8px;
+    padding: 8px;
+    gap: 8px;
+    line-height: 17.8px;
+
+    color: var(--c-text-base-light);
+    font-size: 13.13px;
+    /* font-weight: 700; */
+    font-weight: 510;
+    a {
+    }
```

Comment:
Got a few things up here. Need to remove the comment and the empty `a`, and `510` is an invalid value for `font-weight`. I think we want 500 here.

Reply:
*facepalm* I missed this when I made some other change AND when I did my self-review. Good catch! I'll remove the comment and rule.

I'll go ahead and make that 500. My Firefox inspector wasn't complaining and it does a change when I play around with the 510 value.